torch_np/tests/numpy_tests/core/test_scalarmath.py

```diff
@@ -637,8 +638,8 @@ def test_result(self):
 class TestSubtract:
     def test_exceptions(self):
         a = np.ones((), dtype=np.bool_)[()]
-        with assert_raises(TypeError):
-            operator.sub(a, a)
+        with assert_raises((TypeError, RuntimeError)):    # XXX: TypeError from numpy
+            operator.sub(a, a)                            #  RuntimeError from torch
```

Comment:
Should this be an xfail potentially? Do we want to get in the business of raising the same exceptions at some point or is that outside of the scope?

Reply:
Would be nice to have that at some point, I guess. Finding these in the test suite would be easy though, since this is probably the only reason to catch a tuple of exceptions, so a grep stanza is straighforward.

Reply:
sure, let's do that, but for now, let's document this somewhere so that we don't forget.

Reply:
Opened https://github.com/Quansight-Labs/numpy_pytorch_interop/issues/20